companion-select-text-insert/info.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Inserts selected text from web companion addon and appends current URL below current line ",
+  "identifier": "companion-select-text-insert",
+  "script": "selected-text-below.qml",
+  "authors": ["@v3ss0n"],
+  "platforms": ["linux", "macos", "windows"],
+  "version": "1.0.1",
+  "minAppVersion": " 20.11.7",
+  "description": "This script inserts selected text from webpage and appends the URL below current line.\nRequires Web companion browser extensions:https://github.com/qownnotes/web-companion/"
```

Comment:
I give up and will replace the text myself 😅

Reply:
sorry  , its ok i will do it.

Reply:
pushed with more white space fixes , gonna see if there more lol

Reply:
You'll need to make another pull request.